sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -1461,10 +1478,11 @@ export class CertificateClient {
    *   issuerName: "Self",
    *   subject: "cn=MyCert"
    * });
-   * const pendingCertificate = createPoller.getResult();
+   * let pendingCertificate = createPoller.getResult();
    * console.log(pendingCertificate);
    * const poller = await client.getCertificateOperation("MyCertificate");
-   * const certificateOperation = poller.getResult();
+   * pendingCertificate = poller.getResult();
```

Comment:
Do we need to make pendingCertificate mutable here? Seems like we could do these as two different variables?

Reply:
Oh thanks! this is a mistake.

Reply:
I'm trying to portray that both values should be the same. Is naming them `firstPendingCertificate` and `secondPendingCertificate` better? Or another approach, let me know 🌞

Reply:
I don't think it's clear that the intent is that they should be the same. Is there something specific you want to draw their attention to?